addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -3,15 +3,54 @@
 import { Component } from "@odoo/owl";
 import { useRtc } from "@mail/rtc/rtc_hook";
 import { isMobileOS } from "@web/core/browser/feature_detection";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { _t } from "@web/core/l10n/translation";
 
 export class CallActionList extends Component {
+    static components = { Dropdown, DropdownItem };
     static props = ["thread", "fullscreen", "compact?"];
     static template = "mail.CallActionList";
 
     setup() {
         this.rtc = useRtc();
     }
 
+    get actions() {
+        const acts = []
+        acts.push({
+            id: "raiseHand",
+            name: !this.rtc.state?.selfSession.raisingHand ? _t("Raise Hand") : _t("Lower Hand"),
+            icon: "fa fa-hand-paper-o",
+            onSelect: () => this.rtc.raiseHand(!this.rtc.state.selfSession.raisingHand),
+        });
+        if (isMobileOS) {
+            acts.push({
+                id: "shareScreen",
+                name: !this.rtc.state.sendScreen ? _t("Share Screen") : _t("Stop Sharing Screen"),
+                icon: "fa fa-desktop",
+                onSelect: () => this.rtc.toggleVideo('screen'),
+            });
+        }
+        if (!this.props.fullscreen.isActive) {
+            acts.push({
+                id: "fullScreen",
+                name: _t("Enter Full Screen"),
+                icon: "fa fa-arrows-alt",
+                onSelect: () => this.props.fullscreen.enter(),
+            });
+        }
+        else {
+            acts.push({
+                id: "exitFullScreen",
+                name: _t("Exit Full Screen"),
+                icon: "fa fa-compress",
+                onSelect: () => this.props.fullscreen.exit(),
+            });
```

Comment:
The translated words were moved from `.xml` to `.js`. As a change in stable version, we should adapt the `.pot` file accordingly